client/state/happychat/selectors.js

```diff
@@ -42,15 +42,15 @@ export const getHappychatTranscriptTimestamp = state => (
  * @return {String} current state value
  */
 export const getHappychatConnectionStatus = createSelector(
-	state => state.happychat.connectionStatus
+	state => get( state, 'happychat.connectionStatus' )
```

Comment:
I agree that we really need to turn those into constants. Is it OK to do this in another PR though? Nothing about the logic of those strings is changing in this PR — I'm simply protecting against errors when the state tree is missing keys.